components/Feature/AddAction/index.js

```diff
@@ -72,21 +72,24 @@ const AddAction = ({ onActionAdded }) => {
         </h3>
         <form onSubmit={addToPlan}>
           <TextInput
+            hint={`Example: '${residentName} to provide...' or '${officerName} to contact...'`}
```

Comment:
I see in the screenshot this appears for officers as "Firstname Lastname". Do you think it would be worth trying to grab first name from that string and using that? Something like this might do the trick?

```(js)
officerName.split(' ')[0]
```

This _should_ also work if the officer does not have a last name since they'll be no spaces in that scenario.

Reply:
Yes, that sounds good!